processor/stream/package_tests/metadata_attrs_test.go

```diff
@@ -65,7 +65,8 @@ func TestMetadataPayloadMatchJsonSchema(t *testing.T) {
 	metadataProcSetup().AttrsMatchJsonSchema(t,
 		getMetadataEventAttrs(t, ""),
 		nil,
-		nil)
+		tests.NewSet(tests.Group("user")),
```

Comment:
nitpick: Instead of special handling here, I recommend to add the `user` data to the `only-metadata.ndjson` to have a complete test example.

Reply:
good tip 👍

Reply:
i was thinking that user doesn't make sense in metadata, because it's usually a different user per transaction, but I've added it now

Reply:
Not sure I understand, the [json spec](https://github.com/elastic/apm-server/blob/v2/docs/spec/metadata.json) defines `user`. You are testing that the payload attributes meat the json spec with this test, so I think `user` should be covered here.

Reply:
yeah. My point was i didn't realize we allowed the user in the metadata